### ISSUE TITLE
identifier_prefix appears to be unused

### ISSUE THREAD

**Issue by nergdron** [terraform-aws-modules/terraform-aws-rds-aurora#12]
looking at the docs, I'd expect identifier_prefix to be used in the cluster id, instance id, and elsewhere. however, grepping the code, it seems like it's not used anywhere:

```
$ ag identifier_prefix ../../modules/aws-rds-aurora/
../../modules/aws-rds-aurora/README.md
74:| final_snapshot_identifier_prefix | The prefix name to use when creating a final snapshot on cluster destroy, appends a random 8 digits to name to ensure it's unique too. | string | `final` | no |
75:| identifier_prefix | Prefix for cluster and instance identifier | string | `` | no |

../../modules/aws-rds-aurora/main.tf
27:  final_snapshot_identifier       = "${var.final_snapshot_identifier_prefix}-${var.name}-${random_id.snapshot_identifier.hex}"

../../modules/aws-rds-aurora/variables.tf
10:variable "identifier_prefix" {
53:variable "final_snapshot_identifier_prefix" {
```

**Comment by max-rocket-internet** [comment]
Correct. Feel free to make a PR.

**Comment by bryantbiggs** [comment]
closed with #36
